src/Components/Blazor/Templates/src/content/BlazorWasm-CSharp/Client/BlazorWasm-CSharp.Client.csproj

```diff
@@ -2,8 +2,6 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <OutputType>Exe</OutputType>
```

Comment:
Isn't OutputType=Exe what we want for an application? There does have to be a static Program.Main entrypoint in order for it to run. It's good if the compiler stops you from building without that.

Reply:
I just tried it, and you still get a compilation error in the client project if you don't have an entry point even without `<OutputType>Exe</OutputType>`.

Reply:
Oh, so `Exe` is the default? If so, fine with me!

Reply:
Exe is the default for `.Web`